extensions/amp-iframe/0.1/test/test-amp-iframe.js

```diff
@@ -550,4 +553,16 @@ describe('amp-iframe', () => {
     expect(isAdLike(e(335, 100))).to.be.true;
     expect(isAdLike(e(341, 100))).to.be.false;
   });
+
+  it('should not render fixed ad', () => {
+    return getAmpIframe({
+      src: iframeSrc,
+      sandbox: 'allow-scripts allow-same-origin',
+      width: 300,
+      height: 250,
+      position: 'fixed',
+    }).then(amp => {
+      expect(amp.iframe).to.be.null;
+    }).catch(() => {});
```

Comment:
There's a failure in pollForLayout. The amp-iframe should have '-amp-layout' class but in the test it doesn't. If layoutCallback returns early, there would be no '-amp-layout' class on the element? I don't know how I can handle the pollForLayout failure.